setup.py

```diff
@@ -68,8 +68,12 @@
     ],
     extras_require={
         'rllib': [
+            'GPUtil',
+            'GPy',
             'gym',
             'ray[rllib,default]==1.13.0',
+            'scikit-learn',
+            'torch',
```

Comment:
I think this is unrelated to this change. I see that this is needed for the pettingzoo example (and I'll add it for that). I don't think it's needed for the rllib example?

Reply:
Agreed - removed